source/containers.tex

```diff
@@ -7418,7 +7399,7 @@
     template <class InputIterator> void insert(InputIterator first, InputIterator last);
     void insert(initializer_list<value_type>);
 
-    iterator erase(const_iterator position);
+    iterator  erase(const_iterator position);
     size_type erase(const key_type& k);
     iterator erase(const_iterator first, const_iterator last);
```

Comment:
Shouldn't this third erase() overload be lined up too? It is for unordered_map::erase() but not the other unordered containers.